example/README.md

```diff
@@ -413,7 +415,7 @@ class ShareViewController: SLComposeServiceViewController {
     private func getThumbnailPath(for url: URL) -> URL {
         let fileName = Data(url.lastPathComponent.utf8).base64EncodedString().replacingOccurrences(of: "==", with: "")
         let path = FileManager.default
-            .containerURL(forSecurityApplicationGroupIdentifier: "group.\(hostAppBundleIdentifier)")!
+            .containerURL(forSecurityApplicationGroupIdentifier: this.appGroupId)!
```

Comment:
Here, you cannot use `this`. At the other places, `this` was assigned to `self`. As a result, you should either assign `let this = self` or call `self.appGroupId` here.